ROADMAP.md

```diff
@@ -24,11 +24,13 @@
 
 ## 0.14.0
 
-- [x] Deprecate the old menu folder. #2443
-- [ ] Deprecate usage of JSON in favor of composability.
+- [x] #2443 Deprecate the old menu folder.
+- [ ] #2456 Deprecate usage of JSON in favor of composability.
 - [ ] Documentation versioning (This is a must before removing deprecations).
 - [ ] Enforce eslint rules.
+- [ ] #2433 Auto-generate Docs from source-code.
+- [ ] #2493 Use higher order components across the library to abstract themes passed down from context.
 
 ##### Breaking Changes
```

Comment:
I don't think that we need à breaking changes section is the roadmap, that's for the changelog